pageserver/src/tenant/timeline.rs

```diff
@@ -1747,6 +1747,14 @@ impl Timeline {
             }
         }
 
+        if try_freeze_and_flush {
```

Comment:
isn't the previous block the same `if try freeze_and_flush` condition?  Can we move the code into that block?

Reply:
moved :) didn't notice that after many refactors of this patch... 🤣